### DISCUSSION TITLE
script: Intercepting a request that is too long in time

### DISCUSSION THREAD

**Discussion by 2000ede** [arendst/Tasmota#20380] [Q&A]
Hello,

If I access another ESP32 or PC in the script and it is currently offline, my Tasmota code doesn't really like it.

Example call:

res=http("192.168.178.111" "/cm?&cmnd=status%208")

How can I use a loop to query in advance whether the target address can be reached?
Ping?

if( ..... )

Do you have an example for me?

Thank you!

**Comment by barbudor** [comment]
Tasmota has a PING command (but requires custom build to enable it)
You can probably use it from Scripting although I don't know how (I'm only using it with rules)
